Conference wrap up post for 2024. (#122)
djangocon/2024.djangocon.us: src/_content/posts/conference-wrap-up.md

---
title: "2024 Conference Wrap Up"
layout: post

author: DjangoCon US Communication Team
category: General
published_datetime: 2024-10-07 06:00:00

cover:
    url: /assets/img/blog/Django_group_2024.jpg
    alt: "The DjangoCon US 2024 attendees welcoming everyone to Durham, NC! It's hundreds of people standing with their arms extended outwards."
---

What a fantastic year! DjangoCon US 2024 is officially over, but has left a lasting impression. A heart-felt thank you to each of the 386 folks who attended this year. We had attendees, speakers, and sponsors from all over the world, and we are so thrilled to have such amazing representation from all the people that make up our community.

DjangoCon US is a volunteer-run conference that succeeds because of the dedicated people who give up their lunches, nights, and weekends to tackle tasks small and large. We are very lucky to have so many organizers that see the value this conference brings to the tech world and are willing to donate their time to bring it to fruition. 

> Thank you to the amazing teams we had running our media this year! The folks from American AV, White Coat Captioning and ConFreaks were wonderful to work with. They all provided great suggestions on how we could make our video content better.

Reviewer: ```suggestion
Thank you to the amazing teams we had running our media this year! The folks from American AV, White Coat Captioning, and ConFreaks were wonderful to work with. They all provided great suggestions on how we could make our video content better.
```

Reviewer 2: I think it would be nice lead with "ConFreaks, White Coat Captioning, and ..." 

Why? The order change is based on how long we have worked with these vendors. American AV is relatively new and Confreaks we have worked with for over a decade (they are the only vendor that predates DEFNA being DEFNA).

> We also want to give a special thanks to the speakers, tutorial presenters and community interview team! We were lucky enough to have talks that made us examine both the human and technology ecosystems, while still helping us learn and making us laugh. Keep an eye on the [DjangoCon US YouTube channel](https://www.youtube.com/channel/UC0yY6a79pPY9J0ShIHRf6yw) to catch the ones you missed or to re-watch your favorites from this year and last.

Reviewer: ```suggestion
We also want to give a special thanks to the speakers, tutorial presenters and community interview team! We were lucky enough to have talks that made us examine both the human and technology ecosystems, while still helping us learn and making us laugh. Keep an eye on the [DjangoCon US YouTube channel](https://www.youtube.com/channel/UC0yY6a79pPY9J0ShIHRf6yw) to catch the ones you missed or to re-watch your favorites from this year. Don't forget to look through our archive of talks through the years!
```

> Lastly, we want to thank our sponsors. Without their contributions, DjangoCon US would truly not be possible. Sponsorship funding is how we are able to bring people to the conference who might not otherwise be able to attend, provide food and fun to our attendees, hire amazing media folks to make sure our attendees and speakers have the best possible experience, and post talks to YouTube so that people in our community who are unable to attend can share in the learning. We were sponsored by so many companies that truly care about the DjangoCon US community this year, and we are touched by their support.

Reviewer: ```suggestion
Lastly, we want to thank our sponsors. Without their contributions, DjangoCon US truly would not be possible. Sponsorship funding is how we are able to bring people to the conference who might not otherwise be able to attend, provide food and fun to our attendees, hire amazing media folks to make sure our attendees and speakers have the best possible experience, and post talks to YouTube so that people in our community who are unable to attend can share in the learning. We were sponsored by so many companies that truly care about the DjangoCon US community this year, and we are touched by their support.
```

Slightly awkward wording to me
